### PR TITLE
Removing unnecessary and unbalanced brackets

### PR DESCRIPTION
**Pull request by shigeya**
***
<a href="https://pr-preview.s3.amazonaws.com/shigeya/vc-jose-cose/pull/250.html" title="Last updated on Feb 27, 2024, 11:41 PM UTC (fe8748c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/250/99b8196...shigeya:fe8748c.html" title="Last updated on Feb 27, 2024, 11:41 PM UTC (fe8748c)">Diff</a>

### CODE REVIEW

**Comment by brentzundel** [comment]
Thank you for your keen review. I believe this is definitely something that needs to be fixed, but I suspect it should be fixed in the other direction.
That is, instead of `[[SD-JWT]` becoming `SD-JWT`, the correction should probably be `[[SD-JWT]` becoming `[[SD-JWT]]`
@decentralgabe and @selfissued wdyt?

File: index.html [hunk 1]
```diff
@@ -353,7 +353,7 @@ <h2>Securing JSON-LD Verifiable Credentials with SD-JWT</h2>
           to [[VC-DATA-MODEL-2.0]].
         </p>
         <p>
-          A [=conforming SD-JWT issuer implementation=] MUST use [[SD-JWT] to secure this media type.
+          A [=conforming SD-JWT issuer implementation=] MUST use SD-JWT to secure this media type.
```
Comment (TallTed) [inline index.html:356]:
```suggestion
          A [=conforming SD-JWT issuer implementation=] MUST use [[SD-JWT]] to secure this media type.
```

File: index.html [hunk 2]
```diff
@@ -364,7 +364,7 @@ <h2>Securing JSON-LD Verifiable Credentials with SD-JWT</h2>
           <code>cty</code>.
         </p>
         <p>
-          A [=conforming SD-JWT verifier implementation=] MUST use [[SD-JWT] to verify [=conforming JWS documents=] that use this media type.
+          A [=conforming SD-JWT verifier implementation=] MUST use SD-JWT to verify [=conforming JWS documents=] that use this media type.
```
Comment (TallTed) [inline index.html:367]:
```suggestion
          A [=conforming SD-JWT verifier implementation=] MUST use [[SD-JWT]] to verify [=conforming JWS documents=] that use this media type.
```

File: index.html [hunk 3]
```diff
@@ -2109,7 +2109,7 @@ <h2>Verification Algorithm</h2>
 
     <p>
 When verifying a credential or presentation secured with SD-JWT, the algorithm defined in
-[[SD-JWT] for
+SD-JWT for
```
Comment (TallTed) [inline index.html:2112]:
```suggestion
[[SD-JWT]] for
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (shigeya) [reply index.html:356]: @TallTed Thanks. I did the same first. I'm not sure which way is preferable

**Review by brentzundel — APPROVED** [review]

**Review by decentralgabe — APPROVED** [review]

**Pull request merged**